options/posix/generic/pthread-stubs.cpp

```diff
@@ -213,6 +227,9 @@ int pthread_create(pthread_t *__restrict thread, const pthread_attr_t *__restric
 		MLIBC_MISSING_SYSDEP();
 		return ENOSYS;
 	}
+	new_tcb->stackSize = attr.__mlibc_stacksize;
+	new_tcb->guardSize = attr.__mlibc_guardsize;
+	new_tcb->stackAddr = attr.__mlibc_stackaddr;
```

Comment:
Instead of plainly storing the stack size and address from the attribute, we should store the true values. The current code won't work if no explicit stack address is given in the attribute, will it?

Reply:
You're right, it won't.

Reply:
Fixed.